### DISCUSSION TITLE
Using struct tags with `RowToStructByPos`

### DISCUSSION THREAD

**Discussion by haarts** [jackc/pgx#1682] [Q&A] [answered]
Hi all,

I'm using `pgx` to great effect. But I ran into something. Consider this struct:

```go
type Sheep struct {
  Feet int
  Sound string `db:"-"`
}
```

But this `Sound` field doesn't exist in the database. So I thought I'd add the struct tag and move on. However, it seems this doesn't work when using  `pgx.RowToStructByPos[Sheep]`.

I still get the usual "number of field descriptions must equal number of destinations" error. 
Must I revert to `RowToStructByName`?

**Comment by jackc** [comment, marked as answer]
It didn't support that struct tag. But it is a simple enough change. Should work as of 2f6fcf8eb0c721b9ba23c57d57b0c31207358107.

**Reply by haarts** [reply]
Hahaha! Epic! Thank you.
